docs/Status_Reference.md

```diff
@@ -319,6 +319,20 @@ is defined):
   template expansion, the PROBE (or similar) command must be run prior
   to the macro containing this reference.
 
+## load_cell_probe
+
+The following information is available in the
+[load_cell_probe](Config_Reference.md#load_cell_probe):
+- `last_force`: Returns the last force measured by the probes force
+  sensor. The force readout takes place contineously even if no probe
+  command is running, so this value can e.g. be placed on the display
+  to monitor the extrusion force (if applied to the sensor).
```

Comment:
FWIW, I find the vague reference to extruder pressure to be confusing in a reference.  I think it would be preferable to succinctly state what the parameters report.